README.md

```diff
@@ -290,6 +290,21 @@ require("grapple").setup({
         -- Custom: adds padding around window title
         title_padding = " ",
     },
+
+    --Override default floating window mappings
+    win_mappings = {
```

Comment:
I think the approach of splitting them up might be a good idea. Some of these keymaps are only available for certain windows (i.e. `quickfix` is only used in the tags window. We could do something like:

```lua
mappings = {
  select = "<cr>",
  help = "?",
},

tag_mappings = {
  quickfix = "<c-q>",
},

scope_mappings = { ... },
loaded_mappings = { ... },
```

Then, in the respective hooks, do:

```lua
local mappings = vim.tbl_deep_extend("force", app.settings.mappings, app.settings.tag_mappings)
```

Reply:
what about doing:

```lua
mappings = {
tag = {},
scope = {},
loaded = {},
}
```

To not have `mappings` repeated all the time, but this is just a stylistic change so im fine with whatever

Reply:
aaah, now I understand `mappings` is for all *general* actions, then yeah your suggestion is better